src/isar/mission_planner/echo_planner.py

```diff
@@ -54,20 +52,21 @@ def get_mission(self, mission_id: int) -> Mission:
 
             try:
                 drive_task: DriveToPose = self._create_drive_task(tag_name=tag_name)
+                sensors = [x["sensorTypeKey"] for x in tag["sensorTypes"]]
```

Comment:
For readability, there should be a better name name than `x`.
In general, the naming here isn't great. Consider renaming
`mission_tags` to `plan_items`
`tag` to `plan_item`
`x` to `sensor_type` or something more clear

The format the sensor type is stored on is not the best, so it is a bit hard to work with.